net/uradvd/files/etc/init.d/uradvd

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh /etc/rc.common
+
+START=50
+
+USE_PROCD=1
+
+start_instance() {
+	local cfg="$1" enabled device ifname default_lifetime args
+
+	config_get enabled $cfg 'enabled'
```

Comment:
```suggestion
	config_get_bool enabled $cfg 'enabled' 1
```
I think defaulting to 1 is fine as long as the example config explicitly sets enabled to 0.